src/helpers/helpers.ts

```diff
@@ -0,0 +1,35 @@
+import { Todo } from '../types/Todo';
+
+export function getPreparedTodos(
+  todos: Todo[],
+  query: string,
+  selected: string,
+) {
+  let filteredTodos = todos;
+  const filterTodosByQuery = (title: string) => {
+    return title.toLowerCase().includes(query.toLowerCase());
+  };
+
+  if (query) {
+    filteredTodos = filteredTodos
+      .filter(({ title }) => filterTodosByQuery(title));
+  }
+
+  if (selected) {
+    switch (selected) {
+      case 'active':
+        filteredTodos = filteredTodos.filter(({ completed }) => !completed);
+        break;
+
+      case 'completed':
+        filteredTodos = filteredTodos.filter(({ completed }) => completed);
+        break;
+
+      default:
+        filteredTodos = filteredTodos
+          .filter(({ title }) => filterTodosByQuery(title));
+    }
+  }
```

Comment:
instead of using `'active' 'completed'` you could create some enum (store it in types folder) with all of the options and then reuse them everywhere in your code

(so we would know for sure that if we change 'completed' to 'done' for example - it will change everywhere)

Reply:
Good point, please fix this